apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts

```diff
@@ -560,6 +574,15 @@ export class AdvanceReceiptReportComponent implements AfterViewInit, OnDestroy,
                 this.pageConfiguration.totalPages = response.body.totalPages;
                 this.pageConfiguration.totalItems = response.body.totalItems;
                 this.allReceipts = (this.voucherApiVersion === 2) ? response.body.items : response.body.results;
+
+                this.allReceipts.map(receipt => {
+                    let isSeleted = this.selectedReceipts.filter(selectedReceipt => selectedReceipt === receipt?.uniqueName);
```

Comment:
Consider using `some` method instead of `filter` as it will break the loop at the item it founds the match